tools.yml

```diff
@@ -218,6 +218,11 @@ tools:
   toolshed.g2.bx.psu.edu/repos/bgruening/interproscan/interproscan/.*:
     cores: 8
     mem: 40
+    rules:
+    - id: interproscan_metagenome_input_rule
+      if: input_size > 0.9
+      cores: 10
+      mem: 80
```

Comment:
Is it at all possible to put that to 100 or not really?

Reply:
It would be possible, but we should be as close to real-world as possible